datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactory.kt

```diff
@@ -91,7 +93,13 @@ internal object QuestionnaireItemDatePickerViewHolderFactory :
               }
             }
           )
-          DatePickerFragment().show(context.supportFragmentManager, DatePickerFragment.TAG)
+
+          val selectedDate = questionnaireItemViewItem.singleAnswerOrNull?.valueDateType?.localDate
+          val dateFragment = DatePickerFragment()
+          dateFragment.arguments = bundleOf(REQUEST_BUNDLE_KEY_DATE to selectedDate)
+          dateFragment.show(context.supportFragmentManager, DatePickerFragment.TAG)
```

Comment:
this feels more kotlin-y:

```suggestion
          DatePickerFragment().apply {
            arguments = bundleOf(REQUEST_BUNDLE_KEY_DATE to selectedDate)
          }.show(context.supportFragmentManager, DatePickerFragment.TAG)
```

?

Reply:
yup makes sense, done